### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by kianmeng**
Found via `codespell -S .mypy_cache,.git -L
somme,adresse,adresses,fot,mor,mot,yot,matchs`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ to around 2000 searches per second.
 Check the [documentation](http://addok.readthedocs.org/en/latest/) and a
 [demo](http://adresse.data.gouv.fr/map) with French data.
 
-For discussions, please use the [discourse Geocommun forum](https://forum.geocommuns.fr/c/adresses/addok-le-geocodeur/17). Discussions are mostly French, but English is very welcome. 
+For discussions, please use the [discourse Geocommun forum](https://forum.geocommuns.fr/c/addresses/addok-le-geocodeur/17). Discussions are mostly French, but English is very welcome.
```
Comment (yohanboniface) [inline README.md:17]: This is an URL, so I guess it should not be changed.

**Review by yohanboniface — COMMENTED** [review]
Thanks! Just a small comment :)

**Comment by yohanboniface** [comment]
Thanks :)

**Comment by kianmeng** [comment]
:partying_face: :partying_face: :partying_face: :partying_face: :partying_face:

**Pull request merged**